qemu/qemu.go

```diff
@@ -406,6 +406,9 @@ func (fsdev FSDevice) QemuParams(config *Config) []string {
 		deviceParams = append(deviceParams, fmt.Sprintf(",romfile=%s", fsdev.ROMFile))
 	}
 	if fsdev.Transport.isVirtioCCW(config) {
+		if config.Knobs.IOMMUPlatform {
+			deviceParams = append(deviceParams, fmt.Sprintf(",iommu_platform=%s", "on"))
```

Comment:
`fmt.Sprintf` is not needed, could you please just append `iommu_platform=on` ?

Reply:
Yeah, indeed, there is no variables here, thanks!